src/main/java/com/example/springbootbookshop/mapper/BookMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.example.springbootbookshop.mapper;
+
+import com.example.springbootbookshop.config.MapperConfig;
+import com.example.springbootbookshop.dto.BookDto;
+import com.example.springbootbookshop.dto.CreateBookRequestDto;
+import com.example.springbootbookshop.entity.Book;
+import org.mapstruct.Mapper;
+
+@Mapper(config = MapperConfig.class)
```

Comment:
you cab use this mapper without creating a new MapperConfig class

Reply:
I'd rather have a separate class with configuration settings, I know where to go and change what I need